src/declarations.d.ts

```diff
@@ -1,8 +1,8 @@
-import IPFS from 'ipfs'
+export type { IPFSAPI as IPFSApi } from 'ipfs-core/dist/src/components'
```

Comment:
Is this a bug in the ipfs typs?

Reply:
@oed what exactly? 🙂

Reply:
let's wait for this [issue](https://github.com/ipfs/js-ipfs/issues/3358) to be fixed and we'll revisit types 🙂